config/routes.rb

```diff
@@ -77,6 +77,9 @@
   get "/admin", to: "admin#show"
 
   namespace :api do
+    constraints -> { Rails.application.config.npq_separation[:api_enabled] } do
```

Comment:
maybe we can add this under the npq-separation branch to avoid this being in prod?

Reply:
The `api_enabled` flag for npq separation is already disabled in prod ^ we should probably add a few request specs to cover this constraint though

Reply:
ah ok so not under `npq-separation/` but it's protected anyway, makes sense